dspace-spring-rest/src/test/java/org/dspace/app/rest/WorkflowItemRestRepositoryIT.java

```diff
@@ -419,4 +380,739 @@ public void findBySubmitterTest() throws Exception {
+     *
+     * @throws Exception
+     */
+    public void validationErrorsRequiredMetadataTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community with one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity).withName("Collection 1")
+                .withWorkflowGroup(1, admin).build();
+
+        //2. create a normal user to use as submitter
+        EPerson submitter = EPersonBuilder.createEPerson(context)
+                .withEmail("submitter@example.com")
+                .withPassword("dspace")
+                .build();
+
+        context.setCurrentUser(submitter);
+
+        //3. a workflow item will all the required fields
+        XmlWorkflowItem witem = WorkflowItemBuilder.createWorkflowItem(context, col1)
+                .withTitle("Workflow Item 1")
+                .withIssueDate("2017-10-17")
+                .build();
+
+        //4. a workflow item without the dateissued required field
+        XmlWorkflowItem witemMissingFields = WorkflowItemBuilder.createWorkflowItem(context, col1)
+                .withTitle("Workflow Item 1")
+                .build();
+        String authToken = getAuthToken(admin.getEmail(), password);
+
+        getClient(authToken).perform(get("/api/workflow/worfklowitems/" + witem.getID()))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.status", is(true)))
+                .andExpect(jsonPath("$.errors").doesNotExist())
+        ;
+
+        getClient(authToken).perform(get("/api/workflow/worfklowitems/" + witemMissingFields.getID()))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.status", is(false)))
+                .andExpect(jsonPath("$.errors[?(@.message=='error.validation.required')]",
+                        Matchers.contains(
+                                hasJsonPath("$.paths", Matchers.contains(
+                                        hasJsonPath("$", Matchers.is("/sections/traditionalpageone/dc.date.issued"))
+                                )))))
+        ;
+    }
+
+    @Test
+    /**
+     * Test the update of metadata
+     *
+     * @throws Exception
+     */
+    public void patchUpdateMetadataTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community with one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity).withName("Collection 1")
+                .withWorkflowGroup(1, admin).build();
```

Comment:
If I'm not mistaken, this step is not authorized to edit the item metadata.
All tests for editing metadata in this step should be denied

Reply:
If you look to the enduser functionality you are right but the underline API always allow, both on the legacy workflow than on the configurable workflow any action on the item by "reviewers".
In this PR we are just reproducing the current state of the functionality, the edit will be prevent in the UI in the same way than previous DSpace version.
I agree with you that it will be better to enforce that on the API layer also but I will prefer to discuss that without the pressure of the preview release. We can use the DS-4043 ticket for that

Reply:
I noticed you created DS-4179 for this. I adjusted the ticket to make sure it's clear it's an important security bug

Reply:
ok, as stated this is the current behaviour and it is no so critical imho anyway we have a ticket to deal with it :)